src/opencl_common.c

```diff
@@ -1136,8 +1136,11 @@ static char *get_build_opts(int sequential_id, const char *opts)
 			global_opts = OPENCLBUILDOPTIONS;
 
 	snprintf(build_opts, LINE_BUFFER_SIZE,
-	         "-I opencl %s %s%s%s%s%s%d %s%d %s -D_OPENCL_COMPILER %s",
+	         "-I opencl %s %s%s%s%s%s%s%d %s%d %s -D_OPENCL_COMPILER %s",
 	        global_opts,
+	        options.verbosity > VERB_LEGACY &&
```

Comment:
I think I've been trying to write things like this as `>= VERB_DEBUG` instead, which is the same thing but needs less work if we'd add a verbosity level between LEGACY and DEBUG (for example).

Reply:
VERB_LEGACY is 4. Do you want VERB_DEBUG or VERB_MAX?
```C
#define VERB_DEBUG			6
#define VERB_MAX			5
#define VERB_LEGACY			4
```

Reply:
Oh, right. Well since `-g` is a debugging option, VERB_DEBUG is the logical one, so `>= VERB_DEBUG`.

It's peculiar that max is not the max. It's a little less max than debug o.O